bundles/org.openhab.binding.linky/src/main/resources/OH-INF/thing/thing-linky-remote.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="linky"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="linky">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="enedis"/>
+			<bridge-type-ref id="enedis-web"/>
+			<bridge-type-ref id="my-electrical-data"/>
+		</supported-bridge-type-refs>
+
+		<label>Linky</label>
+		<description>
+			Provides your energy consumption data.
+			In order to receive the data, you must activate your account at
+			https://espace-client-particuliers.enedis.fr/web/espace-particuliers/compteur-linky.
+		</description>
+
+		<channel-groups>
+			<channel-group typeId="linky-remote-main" id="linky-remote-main"/>
+			<channel-group typeId="linky-remote-load-curve" id="linky-remote-load-curve"/>
+			<channel-group typeId="linky-remote-daily" id="linky-remote-daily"/>
+			<channel-group typeId="linky-remote-weekly" id="linky-remote-weekly"/>
+			<channel-group typeId="linky-remote-monthly" id="linky-remote-monthly"/>
+			<channel-group typeId="linky-remote-yearly" id="linky-remote-yearly"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="prmId" type="text" required="true">
```

Comment:
As mentioned in a previous comment, could you make this parameter optional.
If not provided, you will determine it as done today.
This parameter has an interest only for a small part of users who have several Linky counters attached to their account.
It would be cool if most of the users could avoid searching and filling this ID.